config/rbac/role.yaml

```diff
@@ -261,7 +261,7 @@ rules:
 - apiGroups:
   - osp-director.openstack.org
   resources:
-  - overcloudipsets
+  - openstackipset
```

Comment:
Does this and its other references in this file need to be plural?  I just see us using the plural elsewhere.

Reply:
got it in the next rebase.